src/js/dashboard/containers/ClaimsAppealsWidget.jsx

```diff
@@ -0,0 +1,124 @@
+import '../../../sass/claims-status.scss';
+
+import React from 'react';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+import {
+  getAppealsV2,
+  getClaimsV2,
+} from '../../claims-status/actions/index.jsx';
+import { APPEAL_V2_TYPE } from '../../claims-status/utils/appeals-v2-helpers';
+import { scrollToTop, setUpPage } from '../../claims-status/utils/page';
+import ClaimsListItem from '../components/ClaimsListItem';
+import AppealListItem from '../components/AppealsListItem';
+
+class ClaimsAppealsWidget extends React.Component {
+  componentDidMount() {
+    if (this.props.canAccessClaims) {
+      // this.props.getClaimsV2();
+    }
+
+    if (this.props.canAccessAppeals) {
+      this.props.getAppealsV2();
+    }
+
+    if (this.props.claimsLoading && this.props.appealsLoading) {
+      scrollToTop();
+    } else {
+      setUpPage();
+    }
+  }
+
+  renderListItem(claim) {
+    if (claim.type === APPEAL_V2_TYPE) {
+      return <AppealListItem key={claim.id} appeal={claim} name={this.props.fullName}/>;
+    }
+
+    return <ClaimsListItem claim={claim} key={claim.id}/>;
+  }
+
+  render() {
+    const {
+      claimsAppealsCount,
+      claimsAppealsList,
+      claimsLoading,
+      appealsLoading,
+    } = this.props;
+
+    let content;
+    const bothRequestsLoaded = !claimsLoading && !appealsLoading;
+    const bothRequestsLoading = claimsLoading && appealsLoading;
+    const atLeastOneRequestLoading = claimsLoading || appealsLoading;
+    const emptyList = !claimsAppealsList || !claimsAppealsList.length;
+
```

Comment:
This is confusing lol.  Would it possibly be simpler to not concatenate the appeals and claims arrays?